code/game/objects/items.dm

```diff
@@ -927,3 +927,25 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 
 /obj/item/proc/should_stack_with(obj/item/other)
 	return type == other.type && name == other.name
+
+/obj/item/proc/attempt_resist_restraints(mob/living/carbon/user, break_cuffs, effective_breakout_time, silent)
```

Comment:
why isn't this scoped to `/obj/item/restraints` ???

Reply:
would you just duplicate the code on the straight_jacket? Duplicate code isn't ideal, but this also really doesn't need to be scoped to such a low level type